src/lib/OpenEXRCore/openexr_priv_structs.h

```diff
@@ -83,7 +112,7 @@ typedef struct _priv_exr_file_t
     exr_read_func_ptr_t  read_fn;
     exr_write_func_ptr_t write_fn;
 
-    atomic_long file_offset; /**< used when writing */
+    atomic_llong file_offset; /**< used when writing */
     exr_ssize_t file_size;
```

Comment:
Similarly, for exr_ssize_t, I suggest we use the explicit sizes here as well; i.e. int64_t, to avoid the issues we have recently seen on main for 32 bit ARM.

Reply:
well, maybe. some of these are actually meant to change size 32-bit OS -> 64-bit OS, the file I/O in particular. Not 100% sure we should enforce a 64-bit file API or not...

Reply:
In thinking about this a bit more, I am reversing my original statement. The OS disposition (32 vs 64) has nothing to do with the file sizes. So the individual file i/o layers need to handle if the request is too large for the memory system in play (but a 32-bit OS should be able to request individual scanlines of a file that is over 4GB (assuming the scanline also fits).